### PR TITLE
tpcc: add checkCondition6 and checkCondition11

### PR DESCRIPTION
**Pull request by mahjonp**
Add checkCondition6 and checkCondition11.

I found that checkCondition11 always failed after running tpcc run command for a while . PTAL @zhouqiang-cl 

Signed-off-by: mahjonp <junpeng.man@gmail.com>

### CODE REVIEW

**Comment by zhouqiang-cl** [comment]
condition11 is not work for mysql

**Comment by siddontang** [comment]
refer to TPC-C specification,  we can know only condition check 1 - 4 must be demonstrated when running TPC-C. 

All of the checks must be met only after loading data.

**Comment by siddontang** [comment]
so we can separate into two checks - run check, and load check.

**Comment by zhouqiang-cl** [comment]
@siddontang PTAL

File: tpcc/check.go [hunk 1]
```diff
@@ -6,22 +6,34 @@ import (
...
 
 // Check implements Workloader interface
-func (w *Workloader) Check(ctx context.Context, threadID int) error {
+func (w *Workloader) Check(ctx context.Context, threadID int, checkForLoad bool) error {
 	// refer 3.3.2
 	checks := []func(ctx context.Context, warehouse int) error{
 		w.checkCondition1,
 		w.checkCondition2,
 		w.checkCondition3,
 		w.checkCondition4,
```
Comment (siddontang) [inline tpcc/check.go]: seem we still can check all in run except 11

Comment (siddontang) [reply tpcc/check.go]: /cc @zhouqiang-cl

**Pull request merged**